### PR TITLE
[BUG-1664] Resolve Meshery logo spacing and alignment issue

### LINKED ISSUE #1664: Meshery Logo not aligned in large screens
### Description
Meshery Logo is not properly aligned on larger screens (screen sizes greater than 1024px). There is extra spacing in the bottom area of the logo.

### Expected Behavior
The logo should be centered vertically and aligned consistently with other elements.

### Screenshots
meshery-io-navbar-issue

### Enviroment:
 - OS: Linux (Ubuntu)
 - Browser: Chrome
 - Version: Ubuntu 22.04.4 LTS
 - Device: Desktop Mobile

### Note:
I am resolving this issue, kindly assign it to me. Thank you.

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Muhammed-Moinuddin**
Remove extra margin, padding, and add align-items property.

**Description**

This PR fixes #1664

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thanks for jumping in here, @Muhammed-Moinuddin 👍

File: .jekyll-metadata
Comment (ashutosh-rath02) [inline .jekyll-metadata:1]: Please revert this change

**Comment by ashutosh-rath02** [comment]
Hey @Muhammed-Moinuddin Please make the suggested changes and we are good to go.

**Comment by ashutosh-rath02** [comment]
@Muhammed-Moinuddin any updates? Seems you have reverted everything.

**Comment by Muhammed-Moinuddin** [comment]
> @Muhammed-Moinuddin any updates? Seems it has been fixed by #1681. You can close the PR.
Noted. I wasn't feeling well and didn't see it.
